projects/table/src/lib/utils/reset-form.util.ts

```diff
@@ -0,0 +1,73 @@
+import { FormControl, FormGroup } from '@angular/forms';
+
+/**
+ * Resets the NgxTable form if new data is provided, so new controls can be added to the form
+ *
+ * @param currentForm - The form we wish to mutate
+ * @param data - The provided data in the table
+ * @param selectableKey - Optional selectable key
+ * @param resetFormOnNewData - Whether we wish to reset the form when new data was provided
+ */
+export const resetNgxTableForm = (
+	currentForm: FormGroup,
+	data: unknown[],
+	selectableKey?: string,
+	resetFormOnNewData?: boolean
+): void => {
+	// Iben: If no selectableKey was provided but resetFormOnNewData is true, we throw an error. This is because then the form is index based, and it cannot be reset
+	if (!resetFormOnNewData && !selectableKey) {
```

Comment:
Deze lijn en de bijbehorende comment komen niet overeen.
`...but resetFormOnNewData is true, ...` en `if (!resetFormOnNewData ...`.